src/client-scopes/add/MapperDialog.tsx

```diff
@@ -179,12 +179,21 @@ export const AddMapperDialog = (props: AddMapperDialogProps) => {
           variant={TableVariant.compact}
           cells={header}
           onSelect={(_, isSelected, rowIndex) => {
-            rows[rowIndex].selected = isSelected;
-            setRows([...rows]);
+            if (rowIndex === -1) {
+              setRows(
+                rows.map((oneRow) => {
+                  oneRow.selected = isSelected;
+                  return oneRow;
+                })
```

Comment:
Not sure if you are mutating the original object to keep reference equality, but if not you could also do:
```suggestion
                rows.map((row) => ({
                  ...row,
                  selected: isSelected,
                }))
```